src/software/ai/navigator/path_planner/theta_star_path_planner.cpp

```diff
@@ -486,8 +494,10 @@ void ThetaStarPathPlanner::resetAndInitializeMemberVariables(
 {
     // Initialize member variables
     this->obstacles       = obstacles;
-    max_navigable_x_coord = navigable_area.xLength() / 2.0 - ROBOT_MAX_RADIUS_METERS;
-    max_navigable_y_coord = navigable_area.yLength() / 2.0 - ROBOT_MAX_RADIUS_METERS;
+    max_navigable_x_coord =
+        std::max(navigable_area.xLength() / 2.0 - ROBOT_MAX_RADIUS_METERS, 0.0);
```

Comment:
Can rectangle lengths be negative?

Reply:
Oh I just caught up on slack, so xLength might be 0?

Reply:
Yeah, if `xLength / 2.0` is < `ROBOT_MAX_RADIUS_METERS` this results in a negative value.